mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule YamlElixir.Mixfile do
   def project do
     [
       app: :yaml_elixir,
-      version: "1.3.2",
+      version: "1.4.0",
```

Comment:
Since this is a breaking change, shouldn't this be a major version bump?  To quote semver:

>  MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards-compatible manner, and
    PATCH version when you make backwards-compatible bug fixes.

Making this change in a minor version has broken my library that required `{:yaml_elixir, "~> 1.3"}`

Reply:
Oh whoops, sorry about that.

Reply:
@KamilLelonek remedial action here is to bump the version to 2.0.0, push that package, then delete 1.4.0 from hex.pm (if hex allows that, otherwise revert and push 1.3.2 as 1.4.1).